x-pack/solutions/observability/plugins/apm/test/scout/ui/fixtures/page_objects/service_details/alerts_tab.ts

```diff
@@ -30,7 +31,10 @@ export class AlertsTab extends ServiceDetailsTab {
   }
 
   protected async waitForTabLoad() {
-    // Wait for alerts table to load (either empty state or with data)
+    await this.page
+      .getByTestId('showQueryBarMenu')
```

Comment:
```suggestion
  await this.page.testSubj
    .locator('showQueryBarMenu')
```

Just to be consistent with the rest of the file.